### DISCUSSION TITLE
Strict Tracking causes leaderboard ordering to be wrong with Classic scoring

### DISCUSSION THREAD

**Discussion by WitherFlower** [ppy/osu#19229] [Q&A] [answered]
Currently, strict tracking awards two 300 hits on sliders, making it "overweighted" by the classic scoring algorithm.
A solution would be to make sliderends award a LargeTick instead of a full 300 hit when the mod is enabled.

Examples : 
![image](https://user-images.githubusercontent.com/52672543/179711169-b89f6efd-8534-4242-9138-034acda6a4b9.png)
![image](https://user-images.githubusercontent.com/52672543/179711665-cb3956c2-8a50-455c-92c7-53632a10b826.png)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/19232
